### ISSUE TITLE
change tests for 'custom_api' to 'decomposer'

### ISSUE THREAD

**Issue by epogrebnyak** [mini-kep/db#61]
```
_________________ ERROR collecting tests/test_custom_api.py __________________
ImportError while importing test module 'C:\Users\Евгений\Documents\GitHub\mini-kep-db\tests\test_custom_api.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_custom_api.py:2: in <module>
    import db.custom_api.custom_api as custom_api
E   ModuleNotFoundError: No module named 'db.custom_api.custom_api'

```

**Comment by epogrebnyak** [comment]
This is likely blocked by https://github.com/mini-kep/db/issues/63

**Comment by epogrebnyak** [comment]
Closed by 21d09c43a625947f30202630f8f07d8f50fd1e79
